marine.c

```diff
@@ -719,7 +720,7 @@ marine_cf_open(capture_file *cf) {
     cf->epan = marine_epan_new(cf);
 }
 
-WS_DLL_PUBLIC int init_marine(void) {
+WS_DLL_PUBLIC int _init_marine(void) {
```

Comment:
This shouldn't remain a `WS_DLL_PUBLIC` function.
`WS_DLL_PUBLIC` is a wireshark specific macro that means this is a public function that should exported in the shared library.